x-pack/plugins/security/server/plugin.ts

```diff
@@ -187,13 +184,8 @@ export class Plugin {
 
         license,
 
-        // We should stop exposing this config as soon as only new platform plugin consumes it. The only
-        // exception may be `sessionTimeout` as other parts of the app may want to know it.
-        config: {
-          loginAssistanceMessage: config.loginAssistanceMessage,
-          secureCookies: config.secureCookies,
-          cookieName: config.cookieName,
-        },
+        // We should stop exposing this config as soon as only new platform plugin consumes it.
```

Comment:
Note: this may be unnecessary if you remove `cookieName` from the exposed config, see comment below.

Reply:
> Nit: some additional context here would be good.

Great, thanks!

> Note: this may be unnecessary if you remove cookieName from the exposed config, see comment below.

Replied.